### PR TITLE
Updated Missing_History

### PR DESCRIPTION
**Pull request by garimavs**
Through this Pull Request, I'm updating the content of [Missing history](https://galaxyproject.org/support/missing-history/) in the Hub with that of [Finding and working with "Histories shared with me"](https://training.galaxyproject.org/training-material/faqs/galaxy/histories_shared.html) in the GTN.

Kindly suggest any improvements or amendments.

### CODE REVIEW

File: content/support/missing-history/index.md [hunk 1]
```diff
@@ -15,16 +15,7 @@ If you were working with an unregistered account, the History could really be no
...
-When logged into your account, these are located under **User > Histories Shared with Me**.
-
-**Working with _Histories Shared with Me_**
-
-* Make copies to import the History into your account where they can be worked with.
-* Unshare Histories that you no longer want shared with you or that you have already made a copy of.
-* Shared Histories (when copied into your account or not) do count in portion toward your [total account data quota usage](/support/account-quotas/) at Galaxy Main http://usegalaxy.org and most other [public Galaxy servers](/use).
-* Copies of Histories from others count in full toward your total account data quota. Data copied from others is the same as all of your other active/deleted but not permemenently deleted/purged data.
-* More details about how data shared with you, or that you shared with others, functions with respect to account quota usage [are included in this FAQ](/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed).
-* Full details about data privacy features and settings [are included in this FAQ](/learn/privacy-features/).
```
Comment (annefou) [inline content/support/missing-history/index.md:27]: Do we have something on data privacy features and settings in the GTN? (I can't remember!)

Comment (beatrizserrano) [reply content/support/missing-history/index.md:27]: [this](https://github.com/galaxyproject/training-material/pull/3357/)?

File: content/support/missing-history/index.md [hunk 2]
```diff
@@ -15,16 +15,7 @@ If you were working with an unregistered account, the History could really be no
 
 # Someone shared a history with me, where do I find it?
```
Comment (beatrizserrano) [inline content/support/missing-history/index.md:16]:
```suggestion
# [Someone shared a history with me, where do I find it?](https://training.galaxyproject.org/training-material/faqs/galaxy/histories_shared.html)
```

File: content/support/missing-history/index.md [hunk 3]
```diff
@@ -15,16 +15,7 @@ If you were working with an unregistered account, the History could really be no
...
-
-**Working with _Histories Shared with Me_**
-
-* Make copies to import the History into your account where they can be worked with.
-* Unshare Histories that you no longer want shared with you or that you have already made a copy of.
-* Shared Histories (when copied into your account or not) do count in portion toward your [total account data quota usage](/support/account-quotas/) at Galaxy Main http://usegalaxy.org and most other [public Galaxy servers](/use).
-* Copies of Histories from others count in full toward your total account data quota. Data copied from others is the same as all of your other active/deleted but not permemenently deleted/purged data.
-* More details about how data shared with you, or that you shared with others, functions with respect to account quota usage [are included in this FAQ](/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed).
-* Full details about data privacy features and settings [are included in this FAQ](/learn/privacy-features/).
+To find and work on histories shared with you, follow [this FAQ](https://training.galaxyproject.org/training-material/faqs/galaxy/histories_shared.html)
```
Comment (beatrizserrano) [inline content/support/missing-history/index.md:18]:
```suggestion

```

**Review by beatrizserrano — APPROVED** [review]

**Comment by garimavs** [comment]
Thanks @beatrizserrano and @annefou for helping me through this PR!

**Pull request merged**